tests/testlib.c

```diff
@@ -82,6 +88,7 @@ isolated_test_dir_global_setup (void)
   g_assert_cmpstr (g_get_user_cache_dir (), ==, cachedir);
   g_assert_cmpstr (g_get_user_config_dir (), ==, configdir);
   g_assert_cmpstr (g_get_user_data_dir (), ==, datadir);
+  g_assert_cmpstr (g_getenv ("XDG_STATE_DIR"), ==, statedir);
```

Comment:
If you'd run the build-time tests, you'd have noticed this should be `XDG_STATE_HOME`.